src/cubemap.rs

```diff
@@ -95,10 +103,60 @@ impl Cubemap {
         let sampler = unsafe { renderer.device.create_sampler(&sampler_info, None) }
             .map_err(CubemapBuildError::VulkanSamplerCreationFailed)?;
 
+        let folder_path = folder_path.to_owned();
+
+        #[cfg(debug_assertions)]
+        {
+            let ffi_string = std::ffi::CString::new(folder_path.clone())
+                .map_err(|_| CubemapBuildError::InvalidPathConversion(folder_path.clone()))?;
+            let name_info = vk::DebugUtilsObjectNameInfoEXT::builder()
+                .object_handle(final_image.handle.as_raw())
+                .object_type(vk::ObjectType::IMAGE)
+                .object_name(ffi_string.as_c_str());
+
+            unsafe {
+                renderer
+                    .debug_messenger
+                    .as_ref()
+                    .unwrap()
+                    .loader
+                    .set_debug_utils_object_name(renderer.device.handle(), &name_info)
+                    .map_err(CubemapBuildError::VulkanObjectNameAssignationFailed)?
+            };
+
+            let name_info = name_info
+                .object_handle(final_image.view.as_raw())
+                .object_type(vk::ObjectType::IMAGE_VIEW);
+
+            unsafe {
+                renderer
+                    .debug_messenger
+                    .as_ref()
+                    .unwrap()
+                    .loader
+                    .set_debug_utils_object_name(renderer.device.handle(), &name_info)
+                    .map_err(CubemapBuildError::VulkanObjectNameAssignationFailed)?
+            };
+
+            let name_info = name_info
+                .object_handle(sampler.as_raw())
+                .object_type(vk::ObjectType::SAMPLER);
+
+            unsafe {
+                renderer
+                    .debug_messenger
+                    .as_ref()
+                    .unwrap()
+                    .loader
+                    .set_debug_utils_object_name(renderer.device.handle(), &name_info)
+                    .map_err(CubemapBuildError::VulkanObjectNameAssignationFailed)?
+            };
```

Comment:
Maybe regroup the 3 call in a `fn` ?  
```rs
            unsafe {
                 renderer
                     .debug_messenger
                     .as_ref()
                     .unwrap()
                     .loader
                     .set_debug_utils_object_name(renderer.device.handle(), &name_info)
                     .map_err(CubemapBuildError::VulkanObjectNameAssignationFailed)?
             };
```

Reply:
Yeah actually this makes a lot of sense, we should also try to re-use object naming elsewhere, I'll move this to `utils`.

Reply:
Unfortunately, after thinking about it, it didn't make much sense to regroup these 3 operations, in a function. However, https://github.com/TableauBits/morrigu-rs/pull/10/commits/e08fb228991c063cca3e3a07dc54ac07904a5a22 introduces a reusable function to rename vulkan objects, and should help with readability.